alwaysontop/alwaysontop.js

```diff
@@ -0,0 +1,41 @@
+const {
+    api,
+    ondblclick,
+    onload,
+    onbeforeunload
+} = window.alwaysOnTop;
```

Comment:
I'm not sure if there is a guarantee but we are setting it right after window.open. I haven't seen it missing tho. I suggest to leave it like that for now because otherwise we'll have to bring some complexity to the scripts which will slow down displaying the video.